docs/platform.rst

```diff
@@ -737,6 +737,20 @@ In order to use QT, import it from Sgtk::
 Toolkit will make sure Qt is sourced in the correct way. Keep in mind that many applications (for example Nuke)
 may not have a functional Qt that can be imported when they run in batch mode.
 
+When running in an environment with PySide 2, Toolkit will provide under ``sgtk.platform.qt`` an emulation
+layer that mimics the PySide 1 API on top of PySide 2 so that your Toolkit applications don't need to be
+updated to work in those environments. If you need straight access to the PySide 2 API, you can access
+it through the ``sgtk.platform.qt5`` module. Detecting the availability of Qt 5 can be done through
+:meth:`Engine.has_qt5`.
```

Comment:
How difficult would it be to add `Engine.has_qt4` - feels like that could be nice just for symmetry.